client/src/sass/_global.scss

```diff
@@ -3,7 +3,7 @@
 }
 
 html {
-  font-family: 'Raleway', sans-serif;
+  font-family: 'Poppins', sans-serif;
```

Comment:
Personally not my first choice for a display font. Though I also didn't choose Raleway, I do prefer it. Poppins looks OK with the heavier weights.

When all is said and done, I'd like to see whichever display font we use used for just headings anyway–they aren't intended for body text.